samples/drivers/display/sample.yaml

```diff
@@ -169,25 +47,42 @@ tests:
     harness: console
     harness_config:
       fixture: fixture_display
-  sample.display.rk043fn66hs_ctg:
+  sample.display.rk043fn02h_ct:
     platform_allow:
       - mimxrt1064_evk
       - mimxrt1060_evk
       - mimxrt1050_evk
       - mimxrt1040_evk
     tags: display
     harness: console
-    extra_args: SHIELD=rk043fn66hs_ctg
+    extra_args: SHIELD=rk043fn02h_ct
     harness_config:
       fixture: fixture_display
-  sample.display.rk043fn02h_ct:
-    platform_allow:
-      - mimxrt1064_evk
-      - mimxrt1060_evk
-      - mimxrt1050_evk
-      - mimxrt1040_evk
-    tags: display
+  sample.display.shield:
+    # This test case is intended to verify support for shields on boards
+    # known to support them. It is not intended to cover all combinations
+    # of boards and shields, but rather serve as a method to test each
+    # display shield within Zephyr
+    filter: dt_chosen_enabled("zephyr,display")
     harness: console
-    extra_args: SHIELD=rk043fn02h_ct
     harness_config:
       fixture: fixture_display
+    extra_args:
+      - platform:lpcxpresso55s69/lpc55s69/cpu0:SHIELD=adafruit_2_8_tft_touch_v2
+      - platform:nrf52840dk/nrf52840:SHIELD=ssd1306_128x32
+      - platform:frdm_k64f:SHIELD=ssd1306_128x64
+      - platform:mimxrt685_evk/mimxrt685/cm33:SHIELD=waveshare_epaper_gdeh0213b1
+      - platform:nucleo_l433rc_p:SHIELD=waveshare_epaper_gdew042t2
+      - platform:nrf52dk/nrf52832:SHIELD=st7789v_tl019fqv01
+      - platform:lpcxpresso54114/lpc54114/m4:SHIELD=st7789v_waveshare_240x240
+      - platform:frdm_k22f:SHIELD=ls013b7dh03
+      - platform:nrf52833dk/nrf52833:SHIELD=st7735r_ada_160x128
+      - platform:mimxrt1170_evk@B/mimxrt1176/cm7:SHIELD=rk055hdmipi4m
+      - platform:da1469x_dk_pro:DTC_OVERLAY_FILE=da1469x_dk_pro_mipi_dbi.overlay
+      - platform:nrf52840dk/nrf52840:SHIELD=max7219_8x8
+      - platform:stm32h747i_disco/stm32h747xx/m7:SHIELD=st_b_lcd40_dsi1_mb1166
```

Comment:
`sample.display.st_b_lcd40_dsi1_mb1166_a09` above can also be removed & added to this list, it's a variant of `st_b_lcd40_dsi1_mb1166` with a different panel controller.

Reply:
I *think* based on my understanding of https://github.com/zephyrproject-rtos/zephyr/pull/79263, we can only map one shield to a given platform. @hakehuang is this correct? That is why some of the shield cases still exist, because in some cases one platform is being built with multiple shields

Reply:
It is possible, but it would not make sense as this would add both shields to the same build.